kdwsdl2cpp/libkode/style.cpp

```diff
@@ -83,6 +83,9 @@ QString Style::lowerFirst( const QString &str )
 
 QString Style::makeIdentifier( const QString &str )
 {
+  if ( str.isEmpty() )
```

Comment:
Strange place for this check, "" isn't a valid identifier in the first place... No big deal but it makes me wonder, who's calling this with an empty identifier?

Reply:
I don't recall the situation, but it was causing a crash. Thus this was added.

Reply:
Yes the fact that this would crash is obvious, given that at(0) in the implementation.

But it's still an invalid use of this method, in my opinion.
Make it a `Q_ASSERT(!str.isEmpty())` for clarity that this is the contract, but then you'll still have to fix the caller.

Reply:
This is still pending.

Reply:
Shoudl be done now.